### ISSUE TITLE
Webhook com discord nao aparece sobre a partida

### ISSUE THREAD

**Issue by Zackorze** [gamersclub-booster/gamersclub-booster#136]
Adicionei recentemente a extensão e veem me surpreendendo apesar de alguns bugs como ao criar uma lobby e tentar exclui-la acaba bugando dizendo q vc ja esta em uma sendo q nao esta, completar partida nao entendi exatamente e nao sei se esta funcionando. 
  Mas o que venho reportar aqui é sobre o discord que achei uma ferramenta muito boa e útil mas pra mim não tem funcionado os stats da partida, apenas manda sobre a lobby! Muito boa essa extensão, principalmente com o auto-ready q é o q fez mais diferença com a galera aqui q some do pc bem na hr q puxa suhauhasuh continuem assim e se tiver algum jeito de ajudar poderei faze-lo.

![image](https://user-images.githubusercontent.com/99048403/152598432-b1eeac9d-9b56-4dc8-93c9-57fc64efb080.png)

**Comment by crashzk** [comment]
Tenho mesmo problema em relação a falar que já estou em uma lobby mesmo tendo excluída a mesma. Não permiti eu criar de forma alguma, só se desativar a extensão ou esperar 5 minutos para o lobby expirar.

**Comment by Diorv** [comment]
Poxa, o meu também não funciona, queria tanto usar esta função. Tomara que resolvam logo. Abraços!

**Comment by crashzk** [comment]
> Podem informar o navegador/versão e se tem algum momento que isso ocorre em especifico?

Comigo acontece no **Google Chrome - Versão 98.0.4758.82 (Versão oficial) 64 bits**

![image](https://user-images.githubusercontent.com/32937653/153870054-01f9cbf1-4e70-4cc8-855f-1e5d1d7813b7.png)

Acredito que ocorre aleatoriamente, tem momentos que só de eu fechar a lobby e depois tentar criar outra já ocorre isso. Quando acontece isso os players que estão na lobby até falam que a lobby ainda esta lá, eles estão nela, só eu, o criador da lobby que não estou e não consigo entrar nela ou criar outra.

**Comment by henriquelbsouza** [comment]
> > Podem informar o navegador/versão e se tem algum momento que isso ocorre em especifico?
> 
> Comigo acontece no **Google Chrome - Versão 98.0.4758.82 (Versão oficial) 64 bits**
> 
> ![image](https://user-images.githubusercontent.com/32937653/153870054-01f9cbf1-4e70-4cc8-855f-1e5d1d7813b7.png)
> 
> Acredito que ocorre aleatoriamente, tem momentos que só de eu fechar a lobby e depois tentar criar outra já ocorre isso. Quando acontece isso os players que estão na lobby até falam que a lobby ainda esta lá, eles estão nela, só eu, o criador da lobby que não estou e não consigo entrar nela ou criar outra.

valeu, mesmo se tu atualizar a página, continua sem abrir a lobby?

**Comment by crashzk** [comment]
> > > Podem informar o navegador/versão e se tem algum momento que isso ocorre em especifico?
> > 
> > Comigo acontece no **Google Chrome - Versão 98.0.4758.82 (Versão oficial) 64 bits**
> > 
> > ![image](https://user-images.githubusercontent.com/32937653/153870054-01f9cbf1-4e70-4cc8-855f-1e5d1d7813b7.png)
> > 
> > Acredito que ocorre aleatoriamente, tem momentos que só de eu fechar a lobby e depois tentar criar outra já ocorre isso. Quando acontece isso os players que estão na lobby até falam que a lobby ainda esta lá, eles estão nela, só eu, o criador da lobby que não estou e não consigo entrar nela ou criar outra.
> 
> valeu, mesmo se tu atualizar a página, continua sem abrir a lobby?

Sim, mesmo dando F5, fechando e abrindo a página novamente nada resolve. Só aguardando 5 minutos mesmo para finalizar a lobby.

**Comment by leandroribeir0** [comment]
Já aconteceu esse mesmo problema comigo.
Talvez seja uma boa ideia criar um botão para forçar a exclusão da lobby quando dá esse tipo de bug.

**Comment by leandroribeir0** [comment]
> No caso já tem um botão de excluir a sala quando criamos o lobby, porém quando acontece este BUG mesmo usando ele a sala não é deletada, mesmo mostrando que ela foi fechada/excluído.
> 
> Ou seja, você sai da sala mas todos os outros players continuam na lobby, nem mesmo os outros players saindo da lobby a sala é excluida.

Sim, eu sei disso. Quando a gente exclui e acontece o bug a lobby some, se houver um botão para tentar a exclusão da lobby utilizando uma request esse problema pode ser contornado.

**Comment by henriquelbsouza** [comment]
Eu e o @jorgebrunetto estamos tentando resolver.

**Comment by henriquelbsouza** [comment]
Seguinte @crashzk @Zackorze o discord está corrigido no CHROME. Pro firefox eu não consegui resolver. Acho que o próprio firefox tem bloqueio e não faz a request pra API do webhook do discord. Ainda não encontrei uma solução, farei isso quando tiver tempo, sugiro usar chrome, opera ou edge por enquanto.
